File: trivially/src/main/java/trivially/ep/m4/finalized/Neg.java
package trivially.ep.m4.finalized;

import trivially.ep.m4.Exp;

public class Neg implements trivially.ep.m4.Neg {

    protected Exp inner;

    public Exp getInner() {
        return this.inner;
    }

    public void setInner(Exp inner) { this.inner = inner; }

    public Neg(Exp inner) {
        this.inner = inner;
    }
}

File: coco_csharp/CoCo/EP/M5/Finalized/Neg.cs
namespace EP.M5.Finalized {
    public class Neg : Exp, EP.M5.Neg<Exp> {
        private EP.Exp<Exp> inner;

        public Neg(EP.Exp<Exp> _inner) {
            this.inner = _inner;
        }

        public EP.Exp<Exp> getInner() { return this.inner; }
        public void setInner(EP.Exp<Exp> _inner) { this.inner = _inner; }
    }
}